src/core/leaflet-ncwms-autoset-colorscale.js

```diff
@@ -51,8 +50,8 @@ var ncWMSAutoscaleControl = L.Control.extend({
         width: 100,
         height: 100,
       },
-    }).done(function (data) {
-      this.layer.setParams({ colorscalerange: data.min + ',' + data.max });
+    }).then(response => {
+      this.layer.setParams({ colorscalerange: response.data.min + ',' + response.data.max });
```

Comment:
Consider using [template literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals). I find them easier to read and write.